src/Compilers/Core/Portable/Syntax/SyntaxList`1.cs

```diff
@@ -17,9 +19,9 @@ namespace Microsoft.CodeAnalysis
     public readonly partial struct SyntaxList<TNode> : IReadOnlyList<TNode>, IEquatable<SyntaxList<TNode>>
         where TNode : SyntaxNode
     {
-        private readonly SyntaxNode _node;
+        private readonly SyntaxNode? _node;
 
-        internal SyntaxList(SyntaxNode node)
+        internal SyntaxList(SyntaxNode? node)
```

Comment:
>SyntaxNode? [](start = 28, length = 11)

When is this `null`?

Reply:
Most obvious place is here:

```
        public SyntaxList(IEnumerable<TNode>? nodes)
            : this(CreateNode(nodes)!)
```

Which ... I should really remove the `!` from cause it's unnecessary.